LanguageFeatures/Unquoted-imports/lib/co19_unquoted_imports.dart

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion We make a breaking change and remove support for the
+/// long-deprecated library name syntax from `part of` directives. An unquoted
+/// series of identifiers after `part of` then gets unambiguously interpreted as
+/// this proposal's semantics. In other words, `part of foo.bar;` is part of the
+/// library at `package:foo/bar.dart`, not part of the library with name
+/// `foo.bar`.
+///
+/// @description Checks that unquoted syntax can be used in `part` and `part of`
+/// directives.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=unquoted-imports,enhanced-parts
+
+part of co19_unquoted_imports/part_A01_t02;
```

Comment:
Is there a package named `co19_unquoted_imports`? I can't see it here, and I can't see it in https://github.com/dart-lang/co19/tree/master/LanguageFeatures/Unquoted-imports. If there is no such package (presumably, we aren't getting it from `dart pub get`) then this would be a compile-time error. Similarly for all the other unquoted imports that rely on a package with this name.

Reply:
I believe the expanded URI would be `'package:co19_unquoted_imports/part_A01_t02.dart'`, and I don't think that URI could be resolved successfully.

Reply:
OK, now that I've found the pubspec.yaml it looks fine!